tests/jquery.flot.logaxis.Test.js

```diff
@@ -262,15 +262,15 @@ describe("integration tests for log scale functions", function() {
         expect(axes.yaxis.min).toBe(0.0001);
     });
 
-    it('should extend PREFERRED_LOG_TICK_VALUES if too little interval', function() {
+    it('should use EXTENDED_LOG_TICK_VALUES if the interval is too little', function() {
```

Comment:
I don't really like the name of the test - we are testing the behaviour of the code and not the specifics of the implementation.